docs/en/reference/attributes-reference.rst

```diff
@@ -59,12 +148,12 @@ inside the instance variables PHP DocBlock comment. Any value hold
 inside this variable will be saved to and loaded from the database
 as part of the lifecycle of the instance variables entity-class.
 
-Required attributes:
+Required parameters:
```

Comment:
This renames are unrelated, but seemed sane to do, since naming the fields attributes on Attribute classes is redundant and confusing as @greg0ire pointed out.